civihr_employee_portal/src/Blocks/AbsenceLinks.php

```diff
@@ -16,13 +16,14 @@ public function generateBlock() {
         $links = '';
         $links .= '<div class="chr_panel__footer">';
         $links .= '<div class="chr_actions-wrapper">';
+        $links .= civihr_employee_portal_make_link(t('Request TOIL'), 'credit', '', 'pull-left');
```

Comment:
You moved this outside of `.chr_actions-wrapper__inline-duo`, so probably the end result in smaller viewports has changed, please check that